go/utils/slice_utils.go

```diff
@@ -0,0 +1,17 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package utils
+
+// check if sliceA is a subset of sliceB
+func IsSubset(sliceA, sliceB []string) bool {
```

Comment:
Is it a test util? Probably, should be un-capitalized to un-export it.

Reply:
hmm I wasn't able to use this in shared tests without exporting it

Reply:
Can you move to IT dir then?